exporters/otlp/retry.go

```diff
@@ -0,0 +1,63 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package otlp
+
+import (
+	"time"
+
+	"github.com/cenkalti/backoff/v4"
+)
+
+// RetrySettings defines configuration for retrying batches in case of export failure.
+// The current supported strategy is exponential backoff.
```

Comment:
Is there a plan to support other strategies? I feel like we can phrase this as _the_ way we will support backoff.

```suggestion
// RetrySettings defines configuration for retrying batches in case of export failure using an exponential backoff.
```

Reply:
Applied your suggestion here.